Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -27,9 +28,21 @@ public void setupMyCommands(
     public void setupCollectorCommands(
             OperatorInterface oi,
             EjectCollectorCommand eject,
-            IntakeCollectorCommand intake
-    )   {
+            IntakeCollectorCommand intake)
+    {
         oi.leftButtons.getifAvailable(1).whileHeld(eject);
         oi.leftButtons.getifAvailable(2).whileHeld(intake);
     }
+    
+    @Inject
+    public void setupAgitatorCommands(
+            OperatorInterface oi,
+            SpinAgitatorForwardsCommand forwards,
+            SpinAgitatorBackwardsCommand backwards,
+            StopAgitatorCommand stop)
+    {
+        oi.leftButtons.getifAvailable(3).whenPressed(forwards);
+        oi.leftButtons.getifAvailable(4).whenPressed(backwards);
+        oi.leftButtons.getifAvailable(5).whenPressed(stop);
```

Comment:
This is unnecessary. The default command is to stop, so this just takes up an extra button without purpose.